docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -0,0 +1,113 @@
+Run LLMs with OpenVINO GenAI Flavor on NPU
+==========================================
+
+.. meta::
+   :description: Learn how to use the OpenVINO GenAI flavor to execute LLM models on NPU.
+
+This guide provide some extra details on how to utilize NPU with GenAI
+flavor. Please refer to
+:doc:`install OpenVINO GenAI <../../get-started/install-openvino/install-openvino-genai>`
+for more information on OpenVINO GenAI.
+
+Export an LLM model via Hugging Face Optimum-Intel
+##################################################
+
+1. Install the correct components for exporting model:
+   .. code-block:: text
+   
+   # requirements.txt
+   transformers==4.42.4
+   openvino==2024.2.0
+   openvino-tokenizers==2024.2.0
+   nncf==2.11.0
+   optimum-intel @ git+https://github.com/huggingface/optimum-intel.git@439d61f79cf55d5d0b28334f577b6ac3c5ced28f
```

Comment:
Where does this point to? Is there a specific version or tag instead of a hash commit?

Reply:
This is the hash commit from `main` branch that was verified to work with NPU.

In general, we may not specify the exact commit, hopefully it won't break NPU (not guaranteed).
`openvino-notebooks` takes `optimum-intel` just from `main` branch, see:
https://github.com/openvinotoolkit/openvino_notebooks/blob/a99c0ec648fc6414fb5c169e2dd0ef396c71f613/notebooks/llm-question-answering/llm-question-answering.ipynb#L28

`%pip install -q "torch>=2.1" "nncf>=2.7" "transformers>=4.40.0" onnx "optimum>=1.16.1" "accelerate" "datasets>=2.14.6" "gradio>=4.19" "git+https://github.com/huggingface/optimum-intel.git" --extra-index-url https://download.pytorch.org/whl/cpu`

Reply:
can you find a proper fixed version or tag here?